test/bot.test.js

```diff
@@ -124,7 +139,7 @@ describe('Bot', function () {
     const text = 'testmessage';
     const newConfig = { ...config, ircNickColor: false };
     const bot = new Bot(newConfig);
-    const guild = createGuildStub();
+    const guild = this.guild;
```

Comment:
maybe just search replace the usages of `guild` to `guild: this.guild` instead? alternatively you could do `const { guild } = this;` - but might as well just inline the usages.